src/bitcoin-client.js

```diff
@@ -36,17 +41,22 @@ function getAddressFromPublicKey(publicKey) {
 /**
  * Gets the fastest fee rate, regardless of the API used.
  *
- * The Blockstream API responds the fastest fee at "1" block, but Mempool.space
- * does that only for the mainnet. For the testnet, it returns fee rates for
- * "144", "504" and "1008" blocks.
+ * Try, in order:
  *
- * In the case all calls fail for any reason, a fallback fee rate will be used.
+ * - To get the fastest fee from the recommended fees,
+ * - to get the 1-block fee from Blockstream's fee estimates,
+ * - fallback to the default rate.
  */
 const getFastestFee = () =>
```

Comment:
Not at all blocking, however do we cache the result of this anywhere? If not, would perhaps caching it for 2 minutes or so help reduce the load on the API?

Reply:
I guess I can do so in a new PR. Good suggestion!